src/widget/widget.cpp

```diff
@@ -1462,13 +1462,9 @@ void Widget::onGroupNamelistChanged(int groupnumber, int peernumber, uint8_t Cha
         g = createGroup(groupnumber);
     }
 
-    QString name = Nexus::getCore()->getGroupPeerName(groupnumber, peernumber);
     TOX_CHAT_CHANGE change = static_cast<TOX_CHAT_CHANGE>(Change);
     if (change == TOX_CHAT_CHANGE_PEER_ADD)
     {
-        if (name.isEmpty())
-            name = tr("<Unknown>", "Placeholder when we don't know someone's name in a group chat");
-
```

Comment:
Given that toxcore can fail to provide a name, and qTox wraps around it by giving an empty string, I think that it would be better to change `<Unknown>` to `<Empty>` instead of removing it, especially given that toxcore can actually return an empty name.

I.e. as far as toxcore / qTox are concerned, the peer name is "empty" in either of those cases, and it would be good if user was notified about it in some way.